source/opt.d

```diff
@@ -1,31 +1,64 @@
 import std.getopt;
+import std.variant;
+import std.file;
 
-void getoptions(string[] args)
+
+struct Options {
+
+		ulong countEpoch;
+		float mutation;
+		string input;
+		float probability;
+
+		this(string inp, float mutate, ulong  count, float prob)
+		{
+			this.input = inp;
+			this.mutation = mutate;
+			this.countEpoch = count;
+			this.probability = prob;
+		}
+}
+
+class NoInputParams : Exception
```

Comment:
Nicméně já mám určitý pochybnosti ohledně použití výjimek v této aplikaci. @fast4shoot mrkni na můj code review...